packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -268,6 +268,10 @@
   color: $text-danger-disabled;
 }
 
+.headingColor {
+  color: $text-heading;
+}
+
```

Comment:
some duplicate work with @arnoldsandoval-okta over here: https://github.com/okta/odyssey/pull/1176/files#r749664327

Can you two please coordinate on the styles to be changed here? And if <Text /> cannot render h* tags are there better names to be using for these selectors?